xikolo-common/Data/Helper/UserHelper.swift

```diff
@@ -25,17 +25,21 @@ class UserHelper {
         }
     }
 
-    static func getByID(_ id: String) throws -> User? {
+    static func getUser(byId id: String) -> User? {
         let request: NSFetchRequest<User> = User.fetchRequest()
         request.predicate = NSPredicate(format: "id == %@", id)
         request.fetchLimit = 1
-        let users = try CoreDataHelper.executeFetchRequest(request)
-        return users.first
+        do {
+            let users = try CoreDataHelper.executeFetchRequest(request)
+            return users.first
```

Comment:
Wouldn't some question marks be enough here? It's not that nice, but I think you can read the logic faster.

Reply:
do you mean something like this? 
```swift
return CoreDataHelper.executeFetchRequest(request).first
```

Reply:
Kind of, it that works. I though of
```swift
return (try? CoreDataHelper.executeFetchRequest(request))?.first
```
Or without the parenthesis if possible